enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -137,6 +148,44 @@ public void backupCanNotBePerformedOverBackupProtocol() throws Exception
                         "--name=defaultport" ) );
     }
 
+    @Test
+    public void dataIsInAUsableStateAfterBackup() throws Exception
+    {
+        // given database exists
+        Cluster cluster = startCluster( recordFormat );
+
+        // and the database has indexes
+        createIndexes( cluster.getDbWithAnyRole( Role.LEADER ).database() );
+
+        // and the database is being populated
+        AtomicBoolean continueFlagReference = new AtomicBoolean( false );
```

Comment:
Should this start as `true`?

Reply:
Reworded and corrected to `true` for clarity